config/conf.go

```diff
@@ -0,0 +1,97 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package config
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/eclipse-xpanse/policy-man/log"
+	"github.com/spf13/viper"
+	"os"
+)
+
+type Conf struct {
+	Mode string `mapstructure:"mode"`
+
+	Host string `mapstructure:"host"`
+	Port string `mapstructure:"port"`
+
+	ShutdownTimeout int64 `mapstructure:"shutdown_timeout"`
+
+	Log SectionLog `mapstructure:"log"`
+
+	SSL SectionSSL `mapstructure:"ssl"`
+}
+
+type SectionLog struct {
+	Level string `mapstructure:"level"`
+	Path  string `mapstructure:"path"`
+}
+
+type SectionSSL struct {
+	Enable     bool   `mapstructure:"ssl"`
+	KeyPath    string `mapstructure:"key_path"`
+	CertPath   string `mapstructure:"cert_path"`
+	KeyBase64  string `mapstructure:"key_base64"`
+	CertBase64 string `mapstructure:"cert_base64"`
+}
+
+var defaultConf = []byte(`
+mode: release
+host: "localhost" # ip address to bind (default: any)
+port: "8080" # ignore this port number if auto_tls is enabled (listen 443).
```

Comment:
It is better to change the port number to any other value. So that we can run xpanse and policy-man together.  We already have two different ports for terraform-boot and zitadel-local